poly-commit/src/linear_codes/utils.rs

```diff
@@ -114,18 +114,13 @@ pub(crate) fn get_num_bytes(n: usize) -> usize {
 }
 
 #[inline]
-pub(crate) fn hash_column<F, C, H>(array: Vec<F>, params: &H::Parameters) -> Result<C::Leaf, Error>
+pub(crate) fn hash_column<F, H>(array: Vec<F>, params: &H::Parameters) -> Result<H::Output, Error>
 where
     F: PrimeField,
-    C: Config,
     H: CRHScheme,
     Vec<F>: Borrow<<H as CRHScheme>::Input>,
-    C::Leaf: Sized,
-    H::Output: Into<C::Leaf>,
 {
-    H::evaluate(params, array)
-        .map_err(|_| Error::HashingError)
-        .map(|x| x.into())
+    H::evaluate(params, array).map_err(|_| Error::HashingError)
```

Comment:
This method is almost trivial now, maybe let's inline this call?

Reply:
The function is inline already, you mean removing the function entirely?

Reply:
yes, I mean inlining the code itself, not just the call. Sorry I should have been more precise.
So that corresponds to indeed removing this function and replacing with the inner code directly. I think in this case it makes sense, WDYT?